src/api/helpers/formatEntityData.js

```diff
@@ -30,10 +40,33 @@ function getDefaultAlias(entity: object) {
 	};
 }
 
+/**
+ * This is function to get language detail from entity data
+ *
+ * @param {object} entity - Entity datail which is fetched from the database
+ * @returns {string[]} languages - List of languages of entity
```

Comment:
Add that it defaults to an empty array.

You should also go through each @return statement and make sure you clarify what the default result is.